### PR TITLE
Vinegar, Mayonnaise, Ketchunaise

### PR DESCRIPTION
**Pull request by lapatison**
## About the PR 
This PR adds a recipe for already existing in-game vinegar.
Vinegar can be used to create BBQ sauce. Currently, BBQ sauce is the only thing that uses vinegar, so I decided to add another vinegar based drink.

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Added vinegar recipe, it can be created by mixing ethanol and oxygen
- add: Added mayo recipe, it can be created by mixing eggs, oil, and vinegar or lemon juice
- add: Added ketchunaise recipe
- add: Added recipe of Posca - old Roman beverage - that can be created from vinegar, salt, and water

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks.yml [hunk 1]
```diff
@@ -1406,6 +1406,23 @@
...
+  - type: SolutionContainerManager
+    solutions:
+      drink:
+        maxVol: 50
+        reagents:
+        - ReagentId: Posca
+          Quantity: 50
+  - type: Drink
+  - type: Sprite
+    sprite: Objects/Consumable/Drinks/glass_lyellow.rsi
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks.yml:1424]: what is a lyellow

**Review by metalgearsloth — DISMISSED** [review]

File: Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks.yml (hunk 1 above)
Comment (lapatison) [reply Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks.yml:1424]: new sprite with `l`ight `yellow` drink texture

Comment (metalgearsloth) [reply Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks.yml:1424]: You can call it light_yellow

Comment (lapatison) [reply Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks.yml:1424]: understood, ill fix it later today

Comment (lapatison) [reply Resources/Prototypes/Entities/Objects/Consumable/Drinks/drinks.yml]: Done

**Pull request merged**